[2024-05-13T03:28:14.846+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-13T03:28:14.857+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.first_task scheduled__2024-05-12T00:00:00+00:00 [queued]>
[2024-05-13T03:28:14.860+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.first_task scheduled__2024-05-12T00:00:00+00:00 [queued]>
[2024-05-13T03:28:14.860+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-13T03:28:14.864+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): first_task> on 2024-05-12 00:00:00+00:00
[2024-05-13T03:28:14.866+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'first_task', 'scheduled__2024-05-12T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpsq9n6on0']
[2024-05-13T03:28:14.867+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2465) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-13T03:28:14.867+0000] {standard_task_runner.py:91} INFO - Job 11: Subtask first_task
[2024-05-13T03:28:14.867+0000] {standard_task_runner.py:63} INFO - Started process 2466 to run task
[2024-05-13T03:28:14.882+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v4.first_task scheduled__2024-05-12T00:00:00+00:00 [running]> on host f1ddc6e75993
[2024-05-13T03:28:14.906+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mani' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-12T00:00:00+00:00'
[2024-05-13T03:28:14.907+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-13T03:28:14.912+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-13T03:28:14.912+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello world, this is the first task!']
[2024-05-13T03:28:14.915+0000] {subprocess.py:86} INFO - Output:
[2024-05-13T03:28:14.916+0000] {subprocess.py:93} INFO - Hello world, this is the first task!
[2024-05-13T03:28:14.916+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-13T03:28:14.916+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-13T03:28:14.924+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=first_task, run_id=scheduled__2024-05-12T00:00:00+00:00, execution_date=20240512T000000, start_date=20240513T032814, end_date=20240513T032814
[2024-05-13T03:28:14.968+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-13T03:28:14.975+0000] {taskinstance.py:3498} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-05-13T03:28:14.976+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
